spec/factories.rb

```diff
@@ -98,28 +98,117 @@
     candidate
     address_type { 'uk' }
 
-    factory :completed_application_form do
-      support_reference { GenerateSupportReference.call }
+    trait :minimum_info do
       first_name { Faker::Name.first_name }
       last_name { Faker::Name.last_name }
       date_of_birth { Faker::Date.birthday }
+      phone_number { Faker::PhoneNumber.cell_phone }
       first_nationality { 'British' }
       second_nationality { 'American' }
+      address_line1 { Faker::Address.street_address }
+      country { 'GB' }
+      interview_preferences { Faker::Lorem.paragraph_by_chars(number: 100) }
+      safeguarding_issues_status { 'no_safeguarding_issues_to_declare' }
```

Comment:
Maybe we can revisit setting this or get rid of the trait that set this: `with_no_safeguarding_issues_to_declare` as there might be a bit of duplication

Reply:
Good point. Looking at the usage of the trait, I think we can just get rid of it.

Reply:
Removed in https://github.com/DFE-Digital/apply-for-teacher-training/pull/4149/commits/4ed85f5119ca6e6fd6135220c0681b890cb9a67e